src/components/Advertiser/AdvertiserTable.vue

```diff
@@ -273,7 +273,7 @@ const errorStore = useErrorStore()
 const userStore = useUserStore()
 const selectedAdvertise = ref({})
 const filter = ref('')
-const selectedDataType = ref({ label: 'Active', value: 'active' })
+const selectedDataType = ref({ label: 'All', value: 'all' })
```

Comment:
We can make it only see 'Active' but the only issue is that once the advertisement is created it will be in pending state until it's approved by any admin so it will not show under 'active' and for some new user it might create a confusion. Disadvantage is that it will make the performance slower. or we can put some information popup or toast message for clear understanding. i think it will be better then. what do u suggest?

Reply:
We could show both pending and active at the same time than. Don't think it should cause any problem.

Reply:
Yes this sounds good.. sure i will work on it..